src/components/KLabel/KLabel.vue

```diff
@@ -80,6 +79,11 @@ export default defineComponent({
 
     :deep(.k-tooltip) {
       font-weight: 400;
+
+      code {
+        background-color: var(--grey-500);
+        color: var(--white);
```

Comment:
suggestion: can we provide the CSS variable fallback values as well?
```suggestion
        background-color: var(--grey-500, color(grey-500));
        color: var(--white, #fff);
```